src/js/containers/explorer/detail/helpers/explorerQuarters.js

```diff
@@ -150,3 +150,48 @@ export const defaultQuarters = () => {
     const current = mostRecentQuarter();
     return availableQuartersInFY(current.year);
 };
+
+
+export const mostRecentPeriod = () => {
+    // go back 30 days prior to today
+    const todayAdjusted = moment().add(3, 'months');
```

Comment:
Update the comment? Not sure what the logic is we're using here.

Reply:
I'm guesing this should be 

```suggestion
    const todayAdjusted = moment().subtract(30, 'days');
```

Reply:
yeah that's what i get for copy pasting, updated the comment to be more clear